tests/vktracereplay.sh

```diff
@@ -47,13 +47,7 @@ function trace_replay {
 	fi
 }
 
-trace_replay cube "" "--PMB false"
-# Test smoketest with pageguard
-trace_replay smoketest "" "--PMB true"
-# Test smoketest without pageguard, using push constants
-trace_replay smoketest "-p" "--PMB false"
-# Test smoketest without pageguard, using flush call
-trace_replay smoketest "--flush" "--PMB false"
+trace_replay ${VULKAN_SDK}/../examples/build cube "" "--PMB false"
```

Comment:
Let's not require the SDK to test vktrace.

Cube can be built out of Vulkan-Tools.

Reply:
More changes to the organization of the repos is to come, and the current plan is to get rid of submodules. Vulkan-Tools will soon be removed from VulkanTools and dependencies will be found in the SDK if possible, and source dependencies will be referenced by environment variables.

Reply:
Yes, but the plan is for local overrides to take precedence over the SDK.  What you've added looks to be tied very closely to the SDK.

Haven't heard anything about more repo splits but that would be very good to know so we don't waste effort.

Reply:
Okay, I think the best solution here is to reference cube through the path.